documents/src/pages/elements/overlay-menu.md

```diff
@@ -600,26 +615,27 @@ section {
 <section>
   <ef-button cta id="button" aria-haspopup="true">Choose Item</ef-button>
 </section>
-<ef-overlay-menu id="menu">
-  <ef-item type="header">Regions</ef-item>
-  <ef-item value="emea" for="emea" selected>EMEA</ef-item>
-  <ef-item value="apac" for="apac">APAC</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="emea">
-  <ef-item value="spain">Spain</ef-item>
-  <ef-item value="france" disabled>France</ef-item>
-  <ef-item value="united-kingdom" for="united-kingdom" selected>United Kingdom</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="united-kingdom">
-  <ef-item value="cardiff">Cardiff</ef-item>
-  <ef-item value="edinburgh">Edinburgh</ef-item>
-  <ef-item value="london" selected>London</ef-item>
-</ef-overlay-menu>
-<ef-overlay-menu id="apac">
-  <ef-item value="china">China</ef-item>
-  <ef-item value="australia">Australia</ef-item>
-</ef-overlay-menu>
-
+<div class="container">
```

Comment:
This is the second section that wrapper element is required as `item-trigger` event listener is being used here.

Reply:
Updated!